mirgecom/euler.py

```diff
@@ -80,25 +82,18 @@ def _facial_flux(discr, eos, q_tpair, local=False):
         "all_faces."  If set to *True*, the returned fluxes are not projected to
         "all_faces"; remaining instead on the boundary restriction.
     """
-    dim = discr.dim
-
     actx = q_tpair[0].int.array_context
+    dim = discr.dim
 
-    flux_int = inviscid_flux(discr, eos, q_tpair.int)
-    flux_ext = inviscid_flux(discr, eos, q_tpair.ext)
-
-    # Lax-Friedrichs/Rusanov after [Hesthaven_2008]_, Section 6.6
-    flux_avg = 0.5*(flux_int + flux_ext)
-
+    euler_flux = partial(inviscid_flux, discr, eos)
```

Comment:
I meant something more like this:
```suggestion
    def euler_flux(q):
        return inviscid_flux(discr, eos, q) @ normal
```

Reply:
Though this is also has some vector/scalar ambiguity due to `inviscid_flux`...